tools/mysql/schema-update-8.sql

```diff
@@ -4,6 +4,32 @@
 
 # This script upgrade DB schema from version 7 to version 8
 
+CREATE TABLE IF NOT EXISTS groups (
```

Comment:
hi @liyaqin1 Is this table used anywhere in the code?

Reply:
great point.
This table is not in teletraan anymore. Instead, it was moved to rodimus.
I updated the pr and test it again.
The upgrade.sh works well:
```
yaqinli@dev-yaqinli:~/code/teletraan/tools/mysql$ docker exec teletraan_pinterest_mysql /var/teletraan/tools/deploydb.sh
yaqinli@dev-yaqinli:~/code/teletraan/tools/mysql$ docker exec teletraan_pinterest_mysql /var/teletraan/tools/upgrade.sh
Will upgrade schema from version 6 to version 15
upgrading to version 7 with schema-update-7.sql...
upgrading to version 8 with schema-update-8.sql...
upgrading to version 9 with schema-update-9.sql...
upgrading to version 10 with schema-update-10.sql...
upgrading to version 11 with schema-update-11.sql...
upgrading to version 12 with schema-update-12.sql...
upgrading to version 13 with schema-update-13.sql...
upgrading to version 14 with schema-update-14.sql...
upgrading to version 15 with schema-update-15.sql...
Successfully upgraded schema from version 6 to version 15
```